city_metrix/cache_manager.py

```diff
@@ -6,7 +6,7 @@
 from city_metrix import s3_client
 from city_metrix.constants import GeoType, GTIFF_FILE_EXTENSION, GEOJSON_FILE_EXTENSION, NETCDF_FILE_EXTENSION, \
     CSV_FILE_EXTENSION, LOCAL_CACHE_URI, DEFAULT_PRODUCTION_ENV, CIF_CACHE_S3_BUCKET_URI, CTCM_CACHE_S3_BUCKET_URI, \
-    CIF_TESTING_S3_BUCKET_URI
+    CIF_TESTING_S3_BUCKET_URI, FILE_KEY_ADMINBOUND_MARKER, FILE_KEY_URBEXTBOUND_MARKER
```

Comment:
`FILE_KEY_ADMINBOUND_MARKER` and `FILE_KEY_URBEXTBOUND_MARKER` have not been defined in constants